packages/spor-bottom-drawer-react-native/src/BottomDrawer.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { DrawerHandle } from "./DrawerHandle";
+import Modal from "react-native-modal";
+import { Dimensions, ScrollView } from "react-native";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+
+type Props = {
+  children: JSX.Element | JSX.Element[];
+  onClose: () => void;
+  isVisible: boolean;
```

Comment:
Jeg foreslår å endre dette til `isOpen`, da det er det vi bruker ellers i Spor for tilsvarende props.